modules/rostests/kmtests/rtl/RtlStrSafe.c

```diff
@@ -0,0 +1,179 @@
+/*
+ * PROJECT:         ReactOS kernel-mode tests
+ * LICENSE:         GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:         Test for ntstrsafe.h functions
+ * COPYRIGHT:       Copyright 2018 Hern�n Di Pietro <hernan.di.pietro@gmail.com>
+ */
+
+#define KMT_EMULATE_KERNEL
+#include <kmt_test.h>
+#include <ntstrsafe.h>
+#include <ntdef.h>
+#include <ndk/rtlfuncs.h>
+
+#define TESTAPI static void
+
+TESTAPI
+Test_RtlUnicodeStringPrintf()
+{
+    WCHAR Buffer[1024];
+    WCHAR OvrBuffer[1024];
+    WCHAR BufferSmall[2];
+    WCHAR BufferSmall2[7];
+    UNICODE_STRING UsString;
+    const WCHAR FormatStringInts[] = L"%d %d %d";
```

Comment:
Constant strings can be `static`. That is better for speed.

Reply:
`static const` inside a function has no benefits on modern compilers/processors, see e.g. https://softwareengineering.stackexchange.com/a/350532
We're even talking about a test here, so let's not be too picky on such details.